mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/CfpDfipProposalsFaq.tsx

```diff
@@ -0,0 +1,111 @@
+import { ThemedScrollViewV2, ThemedTextV2 } from "@components/themed";
+import {
+  AccordionContent,
+  WalletAccordionV2,
+} from "@components/WalletAccordionV2";
+import { tailwind } from "@tailwind";
+import { translate } from "@translations";
+
+export function CfpDfipProposalsFaq(): JSX.Element {
+  const faqContent: AccordionContent[] = [
+    {
+      title: translate(
+        "components/CfpDfipProposalsFaq",
+        "What is the difference between a DFIP and CFP?"
+      ),
+      content: [
+        {
+          text: translate(
+            "components/CfpDfipProposalsFaq",
+            "DeFiChain Improvement proposal (DFIP) aims to improve the key function(s) on DeFiChain.\nCommunity Funding Proposal (CFP) are community projects that build on top of DeFiChain. All approved CFPs are funded by the Community Development fund."
+          ),
+          type: "paragraph",
+        },
+      ],
+    },
+    {
+      title: translate(
+        "components/CfpDfipProposalsFaq",
+        "How are proposal fees calculated?"
+      ),
+      content: [
+        {
+          text: translate(
+            "components/CfpDfipProposalsFaq",
+            "Proposal fees differ between DFIPs and CFPs. The cost of a DFIP proposal (vote of confidence) will be 50 DFI. Subsequently, a CFP proposal will cost 1% of the requested amount (DFI) or 10 DFI, whichever is larger. Proposal fee are automatically calculated and shown in the confirmation page."
+          ),
+          type: "paragraph",
+        },
+      ],
+    },
+    {
+      title: translate(
+        "components/CfpDfipProposalsFaq",
+        "Where can I find more details for all submitted proposals?"
+      ),
+      content: [
+        {
+          text: translate(
+            "components/CfpDfipProposalsFaq",
+            "You may find all on-chain proposals on DeFiscan, under On-Chain Governance tab."
+          ),
+          type: "paragraph",
+        },
+      ],
+    },
+    {
+      title: translate(
+        "components/CfpDfipProposalsFaq",
+        "Can I vote on proposals using Light Wallet?"
+      ),
+      content: [
+        {
+          text: translate(
+            "components/CfpDfipProposalsFaq",
+            "No, only proposal submissions can be done through Light Wallet. Currently, voting can only be submitted on chain using commands via CLI. Voting commands can be generated on DeFiscan via On-Chain Governance tab > Proposal details > Submit vote.\nYou will need to own a full node wallet and at least one masternode to vote."
```

Comment:
let's use this casing, and let's update all translation `.json` too
```suggestion
            "No, only proposal submissions can be done through Light Wallet. Currently, voting can only be submitted on chain using commands via CLI. Voting commands can be generated on DeFiScan via On-Chain Governance tab > Proposal details > Submit vote.\nYou will need to own a full node wallet and at least one masternode to vote."
```

Reply:
Updated to the correct casing